### DISCUSSION TITLE
EfficientAD batch size

### DISCUSSION THREAD

**Discussion by alexerupakat** [open-edge-platform/anomalib#2240] [Q&A]
Why can't we give a higher batch size for training efficientAD? Why it can only train on batch size =1

**Comment by abc-125** [comment]
Because it is the parameter used in the [paper](https://arxiv.org/pdf/2303.14535).

**Reply by alexerupakat** [reply]
Thanks. But can you help me understand why the paper prefers batch size one?

**Reply by rishabh-akridata** [reply]
@alexerupakat Did you understand why they are using a batch size of 1 in EfficientAD?

**Reply by alexriedel1** [reply]
There is evidence that autoencoders might produce sharper output images if trained on smaller batch sizes, maybe it's because of this.
